OpenRA.Mods.Common/Effects/Missile.cs

```diff
@@ -201,7 +201,10 @@ public void Tick(World world)
 			var shouldExplode = (pos.Z < 0) // Hit the ground
 				|| (dist.LengthSquared < info.CloseEnough.LengthSquared) // Within range
 				|| (info.RangeLimit != 0 && ticks > info.RangeLimit) // Ran out of fuel
-				|| (info.Blockable && world.ActorMap.GetUnitsAt(cell).Any(a => a.HasTrait<IBlocksProjectiles>())) // Hit a wall or other blocking obstacle
+				|| (info.Blockable && world.ActorMap
+					.GetUnitsAt(world.Map.CellContaining(pos))
+					.Any(a => a.TraitsImplementing<BlocksProjectiles>()
+					.Any(b => !b.IsTraitDisabled))) // Hit a wall or other blocking obstacle
```

Comment:
Please add a level of indentation to show that it is inside the previous Any as also requested above.